solcore/analytic_solar_cells/depletion_approximation.py

```diff
@@ -383,6 +395,52 @@ def bc(ya, yb):
     return out
 
 
+def get_J_sc_diffusion_green(xa, xb, g, D, L, y0, S, ph, side='top'):
+    xbL = (xb - xa) / L
```

Comment:
Could you add a `science_reference` within this function indicating where is this theorey described, like what is done in https://github.com/qpv-research-group/solcore5/blob/2ca5ffb3a34e076023ce7604259db07945880c0a/solcore/analytic_solar_cells/depletion_approximation.py#L136 ?

Reply:
Done